cmd/installer/cli/install.go

```diff
@@ -101,7 +101,7 @@ func InstallCmd(ctx context.Context, name string) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			clusterID := metrics.ClusterID()
 			metricsReporter := NewInstallReporter(
-				replicatedAppURL(), flags.license.Spec.LicenseID, clusterID, cmd.CalledAs(),
+				replicatedAppURL(), clusterID, cmd.CalledAs(), os.Args[2+len(args):], flags.license.Spec.LicenseID,
```

Comment:
i feel like `os.Args[2+len(args):]` would panic in our dev env (i don't typically use sudo) or if the user doesn't use sudo (shells into sudo), no?

Reply:
nevermind, i don't think that's true. sudo wouldn't be in the os.Args

Reply:
there was a bug. i fixed this thanks to dryrun